fastify.js

```diff
@@ -636,7 +636,7 @@ function fastify (options) {
 
     if (name === 'onClose') {
       this.onClose(fn)
-    } else if (name === 'onReady') {
+    } else if (name === 'onReady' || name === 'onListen') {
```

Comment:
Either separate else if or also check of onRoute beczse onRoute below is doing the same.